vengine/vengine/components/AnimationComponent.hpp

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+struct AnimationComponent
+{
+	float timer;
```

Comment:
Would be nice with a playback rate float, ex 1.0f is normal and 2.0f is twice as fast